priv/schema/rabbit.schema

```diff
@@ -976,11 +976,11 @@ end}.
 {mapping, "proxy_protocol", "rabbit.proxy_protocol",
     [{datatype, {enum, [true, false]}}]}.
 
-%% Whether to stop the rabbit application if VHost data
-%% cannot be recovered.
+%% Whether to stop the rabbit application if a vhost message
+%% store or recovery fails (or another unrecoverable issue is detected).
```

Comment:
Message store recovery failures on freshly started node or when vhost is just added will not cause application to stop. How can we mention that?